it/test/utils/IntegrationTest.scala

```diff
@@ -150,6 +154,11 @@ trait IntegrationTest extends AnyWordSpec with Matchers with GuiceOneServerPerSu
     .configure(configContentFeatureSwitchOff)
     .build()
 
+  lazy val appWithTailoringOn: Application = GuiceApplicationBuilder()
```

Comment:
in local code search, i could not locate usage of this?

Reply:
It looks like it's not used anywhere and can be removed. When I remove and run local tests, everything is fine. But I would still like you to confirm before I remove it.